site/docs/api-quickstart.md

```diff
@@ -11,6 +11,9 @@ The Hive catalog connects to a Hive MetaStore to keep track of Iceberg tables. T
 ```scala
 import org.apache.iceberg.hive.HiveCatalog
 
+// Set Hive warehouse location here to make it available to Spark
+spark.conf.set("hive.metastore.warehouse.dir", "s3://bucket/path/to/warehouse")
```

Comment:
Typically this will be set through hive-site.xml, and putting hive-site.xml in Spark's conf dir. Not sure if it is a good way to set through Spark's conf?

Reply:
Jerry, thanks very much for your review and comment!
It seems not work if setting it in hive-site.xml and putting hive-site.xml in Spark conf dir. I currently have no idea about the reason, but it seems have something to do with that SparkContext#hadoopConfiguration() does not take hive-site.xml into account, while it only cares the followings:
- core-default.xml
- core-site.xml
- mapred-default.xml
- mapred-site.xml
- yarn-default.xml
- yarn-site.xml
- hdfs-default.xml
- hdfs-site.xml

if I get it correctly.

Reply:
I think the comment should be updated to note that this isn't required if you've already set the warehouse location. This is optional if you haven't already configured Hive in your environment's hive-site.xml.